.github/workflows/msys2_ffmpeg.yml

```diff
@@ -0,0 +1,126 @@
+name: MSYS2 FFmpeg Plugins
+on: [push, pull_request, workflow_call]
```

Comment:
Seems this is another version to `https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/.github/workflows/ffmpeg_plugin.yml`, can we remove the old one to reduce the CI matrix?

Reply:
The following workflow covers ubuntu (linux)
https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/.github/workflows/ffmpeg_plugin.yml

while existing PR cater for windows (msys2), basically adding the support for ffmpeg plugin in windows.

Reply:
The PR workflow mimics the ffmpeg plugin script resides in https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/ecosystem/ffmpeg_plugin/build_ffmpeg_plugin.sh